code/provision/playbooks/inventory.yml

```diff
@@ -1,8 +1,29 @@
+bastion         ansible_host=192.171.139.117
+external-proxy  ansible_host=192.168.3.3
+k8s-master      ansible_host=192.168.3.6
+k8s-worker-1    ansible_host=192.168.3.7
+k8s-worker-2    ansible_host=192.168.3.8
+
 [bastions]
-bastion    ansible_host=192.171.139.117
+bastion
+
+[proxy]
+external-proxy
 
-[proxied]
-proxy      ansible_host=192.168.3.3
+[k8s-cluster]
+k8s-master
+k8s-worker-1
+k8s-worker-2
 
-[insecure]
+[k8s-workers]
+k8s-worker-1
+k8s-worker-2
+
+[proxied:children]
 proxy
+k8s-cluster
+
+[new]
```

Comment:
It is the same at the moment but is a target group for configure-servers.yml to all a subset of servers to be secured while we incrementally build the cluster. Long term we're going to need to dynamically create the inventory.